Cython/Compiler/ExprNodes.py

```diff
@@ -14472,6 +14498,59 @@ def analyse_type_annotation(self, env, assigned_value=None):
 
         return modifiers, arg_type
 
+    def generate_result_code(self, code):
+        pass
+
+    def calculate_result_code(self):
+        if self.string_annotation:
+            return self.string.result()
+        else:
+            return self.expr.result()
+
+    def convert_to_string_annotation(self):
+        self.original_expr = self.expr
+        self.expr = copy.copy(self.string)
+        self.type =self.expr.type
+
+    def _evaluated_annotation_wrap_in_try_catch(self, env):
+        from .UtilNodes import TempResultFromStatNode, ResultRefNode
+        pos = self.expr.pos
+        result = ResultRefNode(pos=pos, type=py_object_type)
+        result.lhs_of_first_assignment = True
+        assignment = Nodes.SingleAssignmentNode(
+            pos,
+            lhs = result,
+            rhs = self.expr
+        )
+        backup_assignment = Nodes.SingleAssignmentNode(
+            pos,
+            lhs = result,
+            rhs = self.string
+        )
```

Comment:
I don't think these are complex enough to benefit from line splitting. (Also, PEP-8 doesn't want spaces around the keyword argument assignments.)
```suggestion
        assignment = Nodes.SingleAssignmentNode(pos, lhs=result, rhs=self.expr)
        backup_assignment = Nodes.SingleAssignmentNode(pos, lhs=result, rhs=self.string)
```